frontend/lib/loc/letter-content.tsx

```diff
@@ -143,6 +160,7 @@ const LetterBody: React.FC<LocContentProps> = (props) => (
     {props.accessDates.length > 0 && <AccessDates {...props} />}
     <Requirements {...props} />
     {props.hasCalled311 && <PreviousReliefAttempts />}
+    {props.workOrderTickets?.length && <WorkOrderTickets {...props} />}
```

Comment:
```suggestion
    {!!props.workOrderTickets?.length && <WorkOrderTickets {...props} />}
```
The `0` shows through when no tickets

Reply:
looks like you beat me to this